tests/envs.py

```diff
@@ -1,20 +1,31 @@
 import logging
 import os
-from six.moves.urllib_parse import urlparse
 
 logging.basicConfig(format='%(name)s %(levelname)s - %(message)s')
 log = logging.getLogger()
 log.setLevel(logging.DEBUG)
 handler = logging.StreamHandler()
 log.addHandler(handler)
 
-ETCD_ENDPOINT = os.getenv('ETCD_ENDPOINT') or 'http://localhost:2379'
-_url = urlparse(ETCD_ENDPOINT)
+ETCD_VER = os.getenv('ETCD_VER') or 'v3.3.0'
 
-protocol = _url.scheme
+ETCD_IMG = 'quay.io/coreos/etcd:' + ETCD_VER
 
-host, port = _url.netloc.split(':')
+DOCKER_PUBLISH_HOST = '127.0.0.1'
 
-ETCD_VER = os.getenv('ETCD_VER') or 'v3.3.0'
 
-ETCD_IMG = 'quay.io/coreos/etcd:' + ETCD_VER
+CERTS_DIR = os.path.join(os.path.dirname(__file__), 'certs')
+CA_PATH = os.path.join(CERTS_DIR, 'ca.pem')
+CERT_PATH = os.path.join(CERTS_DIR, 'client.pem')
+KEY_PATH = os.path.join(CERTS_DIR, 'client-key.pem')
+SERVER_CA_PATH = '/certs/ca.pem'
+SERVER_CERT_PATH = '/certs/server.pem'
+SERVER_KEY_PATH = '/certs/server-key.pem'
+
+NO_DOCKER_SERVICE = True
+try:  # pragma: no cover
+    import docker  # noqa
+    NO_DOCKER_SERVICE = False
```

Comment:
I was intended to mock all the api, so I added the `NO_ETCD_SERVICE` to make test runnable when no etcd server avaliable.

you can just delete this since `NO_DOCKER_SERVICE` always false now